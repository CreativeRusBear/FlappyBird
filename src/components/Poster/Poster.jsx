import React from 'react';
import Logo from '../../assets/images/logo.png';
import classes from './Poster.module.css';

const Poster = (props) => {
    return (
        <div className={classes.Poster}>
            <img src={Logo} alt="Logo"/>
        </div>
    )
}

export default Poster;
