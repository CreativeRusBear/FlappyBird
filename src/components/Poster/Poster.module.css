.Poster {
  width: 100%;
  height: 64px;
  background-color: var(--yellow-color);
  display: flex;
  justify-content: center;
  align-items: center;
}

.Poster img {
  height: 100%;
}

@media screen and (min-width: 768px) {
  .Poster {
    width: 50vw;
    height: 100vh;
  }

  .Poster img {
    width: 100%;
    height: auto;
  }
}
