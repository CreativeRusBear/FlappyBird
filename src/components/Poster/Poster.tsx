import React from 'react';
import Logo from '../../assets/images/logo.png';
import classes from './Poster.module.css';

const Poster = () => {
  return (
    <div className={classes.Poster}>
      <img src={Logo} alt="Logo" loading="lazy"/>
    </div>
  );
};

export default Poster;
