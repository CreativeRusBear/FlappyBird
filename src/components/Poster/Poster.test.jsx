import React from 'react';
import renderer from 'react-test-renderer';
import Poster from './Poster';

describe('<Poster />', () => {
  it('should renders correctly', () => {
    const tree = renderer
      .create(<Poster/>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});

