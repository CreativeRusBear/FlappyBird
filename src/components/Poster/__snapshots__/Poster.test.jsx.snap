// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Poster /> should renders correctly 1`] = `
<div
  className="Poster"
>
  <img
    alt="Logo"
    loading="lazy"
    src="logo.png"
  />
</div>
`;
